### ISSUE TITLE
After updating node version NPM still thinks its up to date

### ISSUE THREAD

**Issue by pocockn** [srs/gradle-node-plugin#192]
When changing the node version in build.gradle say from 4.4.5 to 4.4.7, all tasks should see that node has been updated and run an update. However when I run my build it downloads the new node version but NPM still just says 'up to date', from there I have to run the NPM update independately from the build.

node {
    version = '4.4.5'
}

**Comment by srs** [comment]
Yes, this is a limitation for now. It should respect node version in the future.

**Comment by pocockn** [comment]
Awesome, thank you
